app/views/layouts/application.html.erb

```diff
@@ -10,10 +10,12 @@
 
   <body>
     <nav class = "topnav">
+      <%= link_to "Home", "/" %>
       <%= link_to "All Merchants", "/merchants"%>
       <%= link_to "All Items", "/items"%>
       <%= link_to "Cart: #{cart.total_items}", "/cart" %>
-      <%= link_to "Register", "/register" %>
+      <%= link_to "Register", register_path if !current_user %>
+      <%= link_to "Login", login_path if !current_user %>
```

Comment:
This change will make more sense in the next user story

Reply:
good work getting these buttons in, the log out button will be added in US 16